Makefile

```diff
@@ -42,7 +40,7 @@ py-unit:
 	hack/make-rules/py-test/all.sh
 .PHONY: py-unit
 image-build:
-	go run ./hack/imagebuilder
```

Comment:
I don't think we should name this `image-build` or `imagebuilder`. This repo already contains a different image-builder tool for publishing images using GCB. Also this tool is specific to building Prow images and that isn't clear.

Reply:
renamed to `prowimagebuilder`